CHANGELOG.md

```diff
@@ -1,5 +1,44 @@
 # Changelog
 
+## [1.151.2] - 2024-11-26
+
+### API-Changes
+
+- Deprecate webxdc `descr` parameter ([#6255](https://github.com/deltachat/deltachat-core-rust/pull/6255)).
+
+### Features / Changes
+
+- AEAP: Check that the old peerstate verified key fingerprint hasn't changed when removing it.
+- Add `AccountsChanged` and `AccountsItemChanged` events ([#6118](https://github.com/deltachat/deltachat-core-rust/pull/6118)).
+- Do not use format=flowed in outgoing messages ([#6256](https://github.com/deltachat/deltachat-core-rust/pull/6256)).
+- Add webxdc limits api.
+- Add href to IncomingWebxdcNotify event ([#6266](https://github.com/deltachat/deltachat-core-rust/pull/6266)).
+
+### Fixes
+
+- Revert treating some transient SMTP errors as permanent.
+
+### Other
+
+- Prefer long options in RELEASE.md ([#6136](https://github.com/deltachat/deltachat-core-rust/pull/6136))
+
+using long options make things less mystical, it is clearer what
```

Comment:
How did the full commit message get here?

Reply:
good question - i removed that completely, it is not important enough for a changelog

Reply:
good to know, so when one forgets about the prefix, one has to deal with it afterwards :)